benchbuild/source/git.py

```diff
@@ -0,0 +1,124 @@
+"""
+Declare a git source.
+"""
+import typing as tp
+
+import attr
+import plumbum as pb
+from plumbum import local
+
+from benchbuild.utils.cmd import git, mkdir
+from benchbuild.utils.path import flocked
+
+from . import base
+
+if tp.TYPE_CHECKING:
+    from mypy_extensions import VarArg
```

Comment:
Codacy found an issue: [Unable to import 'mypy_extensions'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)